bedrock/firefox/templatetags/helpers.py

```diff
@@ -11,7 +11,7 @@
 from bedrock.firefox.models import FirefoxOSFeedLink
 from bedrock.firefox.firefox_details import firefox_desktop, firefox_android, firefox_ios
 from bedrock.base.urlresolvers import reverse
-from lib.l10n_utils import get_locale
+from lib.l10n_utils import get_locale3
```

Comment:
@JamesDLorence looks like this is a typo. Tests are complaining that it can't import this.